### DISCUSSION TITLE
How to prevent manjaro-sway-theme from overwriting custom foot.ini?

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#683] [Q&A] [answered]
I have made some changes to `~/.config/foot/foot.ini`, like adjusted font-size and some custom key-bindings.

Running `manjaro-sway-theme` will overwrite `~/.config/foot/foot.ini` and thus remove my customization.

Of course, there is the backup file `foot.ini~` and I can easily revert the changes. But if there were changes to the theme colors I would have to merge the changes from the new `foot.ini` and the backup file.

Would it make sense to add an additional file like `foot-theme.ini`, and have `foot.ini` include only that by default? `manjaro-sway-theme` would overwrite `foot-theme.ini`, and users could add their additional changes to `foot.ini`, as before?

**Comment by boredland** [comment]
Sorry that fell under the rug. I think adding another specialist file for that is a good option.

**Reply by heapifyman** [reply]
I could try to create a PR for that.

**Comment by boredland** [comment, marked as answer]
fix is in `manjaro-sway-settings-git` - lets check if that works.
